# Specific view of the plant bed

uint8 bed_id    # ID of the bed
uint8 bed_side  # Side of the bed (0: left, 1: right)